index.html

```diff
@@ -0,0 +1,201 @@
+
+## The `@using` directive
+
+### `@using` within a document
+
+```graphql
+#::[Declaration of the `@using` directive]
+directive @using(spec: using_URL!)
+  repeatable on SCHEMA
+```
+
+The `@using` directive is applied to `schema` declarations to declare that the schema uses a particular spec. For example,
+
+```graphql
+#::[Using the `@using` directive]
+schema @using(spec: "https://spec.example.com/exampleSpec/1") {
+  query: Query
+}
+```
+
+### Requiring `@using` within a spec
+
+Specifications **may** include the following language to instruct schema authors to use `@using` to indicate documents which are intended to conform to the specification:
+
+> SPEC_NAME schema documents **must**/**should**/**may** have a `schema` declaration with a [`@using` directive](/specs/use)
+> and the specification URL `SPEC_URL`:
+>
+> ```graphql
+> schema @using(spec: SPEC_URL) {
+>   query: Query
+> }
+> ```
+
+`SPEC_NAME` and `SPEC_URL` should be replaced with the names and URLs for the specification. For example:
+
+> Example Spec schema documents **must** have a `schema` declaration with a [`@using` directive](/specs/use)
+> and the specification URL `https://specs.example.com/exampleSpec/1`:
+>
+> ```graphql
+> schema @using(spec: "https://specs.example.com/exampleSpec/1") {
+>   query: Query
+> }
+> ```
+
+## Processing Schemas
+
+```mermaid
+#::[Example processing pipeline]
+graph LR
+  schema(["📄  Input Schema"]):::file-->proc("🤖 &nbsp;Processor")
+  proc-->output(["📄  Output Schema"]):::file
+  classDef file fill:none,color:#eee;
+  style proc fill:none,stroke:fuchsia,color:fuchsia;
+```
+
+A common case is that of a processor which consumes an input schema and generates an output schema.
+
+The general guidance for processor behavior is: don't react to what you don't understand.
+
+Specifically, processors:
+  - **should** pass through `@using` directives which reference unknown spec URLs
+  - **should** pass through unknown directives, types, and other schema elements
+  - **should** ignore meaningless URL components (such as unknown query parameters or fragments) within `@using` URLs
+
+An exception to this is processors which prepare the schema for final public consumption. Such processors **may** choose to eliminate all unknown directives and prefixed types in order to hide schema implementation details within the published schema. This will impair the operation of tooling which relies on these directives—such tools will not be able to run on the output schema, so the benefits and costs of this kind of information hiding should be weighed carefully on a case-by-case basis.
+
+## Spec URLs
+
+```graphql
+#::[Declaration of using_URL scalar]
+scalar using_URL
+  @specifiedBy(url: "https://apollographql.com/docs/specs/using#spec-urls")
```

Comment:
Make sure this URL gets updated once we finish painting the bikeshed.

While I realize this isn't a `@using` directive, it seems like this would be a good URL to apply semver to...

Reply:
I think dropping the scalar here was a good idea. It makes it easier to actually use `@using`.

Reply:
yeah.